benchmarking-cli/src/main.rs

```diff
@@ -0,0 +1,159 @@
+use clap::Parser;
+use miden::{Assembler, ProgramInputs, ProofOptions};
+use winter_math::StarkField;
```

Comment:
I think we'll want to revisit this import in the future to get it from somewhere in Miden instead, but we can leave it this way for now.